src/content/guides/farmlands-portal-introduction.mdx

```diff
@@ -193,6 +193,58 @@ On the Settings page, you can change your business information. Please note this
 
 ![Centrapay business portal settings page](/centrapay-portal-settings.png)
 
+## FAQs
+
+- **What do we do if a customer returns one or more items, and the invoice has already been sent to Farmlands?**
+
+    Raise a credit note as normal and insert the original authorisation short code on the invoice. This will enable us to identify the original invoice and customer account details.
+
+- **What do we do if the customer cancels their order after we have sent the invoice to Farmlands?**
+
+    Raise a credit note as normal and insert the original authorisation short code on the invoice. This will enable us to identify the original invoice and customer account details.
+
+- **What do we do if the customer cancels their order before**
```

Comment:
Is this question incomplete?

Reply:
Yes it is, nice spot - I'll find out what it's meant to be